### PR TITLE
[Thematic communities] Add space in front

### PR DESCRIPTION
**Pull request by kevin-verschaeve**
[ch12353]

### CODE REVIEW

File: src/Form/ThematicCommunityMembershipFilterType.php [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+                'required' => false,
+                'choices' => [
+                    'Adhérent',
+                    'Animateur local',
+                    'Porteur de projets citoyens',
+                    'Référent',
+                    'Contact',
+                ],
+                'choice_label' => function ($choice) {
+                    return $choice;
```
Comment (ottaviano) [inline src/Form/ThematicCommunityMembershipFilterType.php:49]: useless no?

File: src/Form/ThematicCommunityMembershipFilterType.php [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+            ->add('ageMin', IntegerType::class, ['required' => false])
+            ->add('ageMax', IntegerType::class, ['required' => false])
+            ->add('city', TextType::class, ['required' => false])
+            ->add('country', CountryType::class, ['required' => false])
+            ->add('role', ChoiceType::class, [
+                'required' => false,
+                'choices' => [
+                    'Adhérent',
+                    'Animateur local',
+                    'Porteur de projets citoyens',
```
Comment (ottaviano) [inline src/Form/ThematicCommunityMembershipFilterType.php:44]: what about using `key => label` structure to be able to modify the only labels in the future and do not touch the keys?

File: src/Form/ThematicCommunityMembershipFilterType.php [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+                    'Oui' => 1,
+                ],
+            ])
+            ->add('job', ChoiceType::class, [
+                'required' => false,
+                'expanded' => false,
+                'multiple' => true,
+                'choices' => JobEnum::JOBS,
+                'choice_label' => function (string $choice) {
+                    return $choice;
```
Comment (ottaviano) [inline src/Form/ThematicCommunityMembershipFilterType.php:120]: same here, it's the default behavior

File: src/Form/ThematicCommunityMembershipFilterType.php [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+            ->add('sort', HiddenType::class, ['required' => false])
+            ->add('order', HiddenType::class, ['required' => false])
+        ;
+
+        if (\count($options['handled_communities']) > 1) {
+            $builder->add('thematicCommunities', EntityType::class, [
+                'required' => false,
+                'class' => ThematicCommunity::class,
+                'multiple' => true,
+                'query_builder' => function (EntityRepository $er) use ($options) {
```
Comment (ottaviano) [inline src/Form/ThematicCommunityMembershipFilterType.php:142]: why do you not use the CT objects from `$options['handled_communities']` directly?
```php
'choices' => $options['handled_communities'],
```

File: src/Repository/ThematicCommunity/ThematicCommunityMembershipRepository.php [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+        if (null !== ($withAssociation = $filter->isWithAssociation())) {
+            $qb->andWhere('tcm.association = :with_association')
+                ->setParameter('with_association', $withAssociation)
+            ;
+        }
+
+        return $qb;
+    }
+
+    public function countMembershipsInCommunities(array $thematicCommunities)
```
Comment (ottaviano) [inline src/Repository/ThematicCommunity/ThematicCommunityMembershipRepository.php:262]: typehint missing

File: src/Twig/DateRuntime.php [hunk 6]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Twig;
+
+class DateRuntime
```
Comment (ottaviano) [inline src/Twig/DateRuntime.php:5]: `implements RuntimeExtensionInterface`

File: translations/messages.fr.yml [hunk 7]
```diff
@@ -1028,6 +1028,10 @@ admin.thematic_community_membership.type.adherent: Adhérent
 admin.thematic_community_membership.type.electedRepresentative: Élu
 admin.thematic_community_membership.type.contact: Contact
 
+thematic_community_chief.popup.title: Charte de bonne utilisation des outils numériques
+thematic_community_chief.popup.subtitle: à l’usage des Responsables Communautés Thématiques
+thematic_community_chief.popup.file_url: https://storage.googleapis.com/en-marche-fr/TECH/charte_numerique_responsables_ct.pdf
```
Comment (kevin-verschaeve) [inline translations/messages.fr.yml:1033]: I need someone to create this file for me :)

File: src/Form/ThematicCommunityMembershipFilterType.php (hunk 3 above)
Comment (kevin-verschaeve) [reply src/Form/ThematicCommunityMembershipFilterType.php:120]: no, as the array only have values, if I don't add this, the choice displayed to the user will be `0, 1, 2, ...`

File: translations/messages.fr.yml (hunk 7 above)
Comment (ottaviano) [reply translations/messages.fr.yml:1033]: done

Comment (kevin-verschaeve) [reply translations/messages.fr.yml:1039]: thanks

**Review by ottaviano — APPROVED** [review]

**Pull request merged**